backend/src/test/java/ai/verta/modeldb/ExperimentRunTest.java

```diff
@@ -731,12 +728,9 @@ public void b_getExperimentFromProjectRunNegativeTest() {
     }
 
     getExperiment = GetExperimentRunsInProject.newBuilder().setProjectId("sdfdsfsd").build();
-    try {
-      experimentRunServiceStub.getExperimentRunsInProject(getExperiment);
-      fail();
-    } catch (StatusRuntimeException e) {
-      checkEqualsAssert(e);
-    }
+    GetExperimentRunsInProject.Response response =
+        experimentRunServiceStub.getExperimentRunsInProject(getExperiment);
+    assertEquals("Expected response not found", 0, response.getExperimentRunsCount());
```

Comment:
Now we are checking permission first so if given project id is not valid it will return empty list instead of PERMISSION_DENIED